README.md

```diff
@@ -29,7 +29,14 @@ cfzone need two environment variables:
 - `CF_API_KEY` - Your API key from [Cloudflare](https://support.cloudflare.com/hc/en-us/articles/200167836-Where-do-I-find-my-Cloudflare-API-key-)
 - `CF_API_EMAIL` - Your Cloudflare email address.
 
-An optional `-yes` flag will cause cfzone to continue syncing without confirmation.
+Run cfzone as with the following command:
+`cfzone <zonefile> [-leaveunknown] [-yes]`
+
+Available optional flags:
+`-leaveunknown`
+	  Don't delete unknown records
+`-yes`
+	  will cause cfzone to continue syncing without confirmation.
```

Comment:
How about making this into a table? This will render as one line.

```
| Flag   | Description |
|--------|-------------|
| `-yes` | Don't ...   |
```

That would render like this:

| Flag   | Description |
|--------|-------------|
| `-yes` | Don't ...   |

Reply:
Great idea... will fix right away :)